src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,60 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class StringTest {
+
+    @DisplayName("split() 사용")
+    @Test
+    void split(){
+
+        String given = "1,2";
+
+        String[] result = given.split(",");
+
+        assertThat(result).contains("1");
+        assertThat(result).containsExactly("1","2");
+    }
+
+
+    @DisplayName("substring() 사용")
+    @Test
+    void substring(){
+        String given = "(1,2)";
+
+        String result = given.substring(1, given.length()-1);
+
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @DisplayName("정상적인 특정 위치에서 문자 가져오기")
+    @Test
+    void charAt(){
+        String given = "abc";
+
+        char resultA = given.charAt(0);
+        assertThat(resultA).isEqualTo('a');
+
+
+        char resultB = given.charAt(1);
+        assertThat(resultB).isEqualTo('b');
+
+        char resultC = given.charAt(2);
+        assertThat(resultC).isEqualTo('c');
```

Comment:
```suggestion
        char resultA = given.charAt(0);
        char resultB = given.charAt(1);
        char resultC = given.charAt(2);
        
        assertThat(resultA).isEqualTo('a');
        assertThat(resultB).isEqualTo('b');
        assertThat(resultC).isEqualTo('c');
```
검증부와 실행부를 나누어보았을때 `split`에서 테스트하신 것처럼 선언 위치를 구분한다면 조금 더 가독성 높은 코드가 될 수 있을 것 같아요 :)